### PR TITLE
change "invalid encryption key" to "invalid serial key"

### PR DESCRIPTION
**Pull request by bhrayhaw**
## Description

The function was pertaining to serialTobyte so I think the output of the should rather be 
"invalid serial key" instead of "invalid encryption key".

## Related Issue

Resolves #[issue_number]

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation
- [ ] Chore (Related to CI or Packaging to platforms)

### CODE REVIEW

**Review by profclems — APPROVED** [review]

File: internal/ecrs/aes.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func SerialFromString(s string) (Serial, error) {
 // SerialFromBytes creates secret from bytes slice.
 func SerialFromBytes(b []byte) (Serial, error) {
 	if len(b) != 16 {
-		return Serial{}, ErrAES.New("invalid encryption key")
+		return Serial{}, errs.New("invalid serial key")
```
Comment (profclems) [inline internal/ecrs/aes.go:89]: Let's wrap the error with `ErrAES` like:
```go
ErrAES.New("invalid serial key")
```

**Review by profclems — CHANGES_REQUESTED** [review]

**Review by profclems — APPROVED** [review]
LGTM!

File: internal/ecrs/aes.go (hunk 1 above)
Comment (bhrayhaw) [reply internal/ecrs/aes.go]: Done resolving

**Review by Aeswolf — APPROVED** [review]
Looks clean.

**Pull request merged**